main/com.tunein/dataset.tt

```diff
@@ -0,0 +1,25 @@
+dataset @com.tunein language "en" {
+    query (p_query :String)  := @com.tunein.search_channel(query=p_query, count=p_count)
```

Comment:
You didn't declare the `p_count` parameter for this template so you can't use it.